### DISCUSSION TITLE
Can github copilot understand other natural languages besides english?

### DISCUSSION THREAD

**Discussion by nicewinter** [community/community#16845] [Copilot Conversations] [answered]
Since it's a descendant of GPT-3, which was trained on many natural languages not limited to English, i would expect copilot can understand German, Spanish, Italian and more - but didn't see it's mentioned anywhere in github official documentation whether or not it can understands comments in other natural languages, or currently it is English only?

**Comment by myuanz** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/25359546/169099712-c8c17624-f539-4b70-b313-901209cf0eae.png)
![image](https://user-images.githubusercontent.com/25359546/169100202-1800015b-f7c5-46ef-80f8-86dc72e58422.png)

English translation of the picture above: 
```
ME: Can you understand Chinese?
AI: I also understand. 

ME: Can you understand Greek?
AI: I understand Greek. 
```

`Write a quick sort and give a test case` (快速排序 -> quick sort): 
![快排](https://user-images.githubusercontent.com/25359546/169102260-377c91d9-1db1-4e9b-b6ac-024c2efd6a82.gif)
